assets/src/block-templates/side-image-with-text-and-cta/block.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "planet4-block-templates/side-image-with-text-and-cta",
+  "title": "Side image with text and CTA",
+  "description": "Side image with text and CTA",
+  "category": "planet4-block-templates",
+  "textdomain": "planet4-blocks-backend",
+  "attributes": {
+    "title": { "type": "string" },
+    "titlePlaceholder": { "type": "string" },
+    "backgroundColor": { "type": "string" },
+    "alignFull": { "type": "bool" },
+    "mediaPosition": { "type": "string" }
```

Comment:
Do you know if it's possible to define somethink like `""mediaPosition: { "type": "right" | "left" }` like in Typescript?

Reply:
Yes good point, [the schema includes an `enum` attribute](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-attributes/), I'm gonna switch to it :+1: